### ISSUE TITLE
Empty images alt attribute on report page

### ISSUE THREAD

**Issue by chteuchteu** [munin-monitoring/munin#332]
An issue has been introduced in the latest version of munin (available for preview at [here](http://demo.munin-monitoring.org/munin/munin-monitoring.org/demo.munin-monitoring.org/)).
As you can see (by displaying page source or inspecting the element in Chrome/Firefox), the alt attribute has en empty value on each graph <img>:

``` html
<img class="i" src="/munin/../munin-cgi-graph/munin-monitoring.org/demo.munin-monitoring.org/if1sec_lo-day.svg" alt="">
```

Previously ([demo still available here](http://demo.munin-monitoring.org/munin-monitoring.org/demo.munin-monitoring.org/)), it was filled:

``` html
<img class="i" src="/munin-cgi/munin-cgi-graph/munin-monitoring.org/demo.munin-monitoring.org/if1sec_eth0-day.png" alt="Interface 1sec stats for eth0">
```

Faulty version is Munin version 2.1.10-debian-auto-2015-01-20-c120-g79bd1cb

**Comment by ssm** [comment]
The munin cgi scripts have been replaced by munin-httpd in the development branch.

**Comment by ssm** [comment]
This comment-and-close does not make sense on this issue, sorry.  Reopening.

**Comment by chteuchteu** [comment]
This issue seems to be fixed (by 2ac292e?). Anyone can confirm?
Alt attributes are well populated, and I haven't found any page where the plugin name isn't displayed.
